app/helpers/form_helper.rb

```diff
@@ -0,0 +1,10 @@
+module FormHelper
+  def bool_to_int bool
```

Comment:
Do you think FormHelper is the place for this?  It might be.  It could also be `class Object; def bool_to_int; self ? 1 : 0`  See `https://github.com/theB264/osra/blob/4914bf9d25f546753b7d191ed139895cb2509366/spec/ruby_extensions/integer_spec.rb` and `https://github.com/theB264/osra/blob/4914bf9d25f546753b7d191ed139895cb2509366/config/initializers/integer_extension.rb`  And that could allow you to call `f.object.request_fulfilled.bool_to_int.to_s`